frontend/src/components/SectionLoader/SectionLoader.scss

```diff
@@ -0,0 +1,79 @@
+// Based on https://loading.io/css/
```

Comment:
Use a pure CSS spinner because
* I'm not sure how to add static images to this repo
* This spinner has a friendly license (Creative Commons CC0).